container/conf.d/daemon.conf

```diff
@@ -0,0 +1,18 @@
+[daemon]
+systemd_enable = false
+
+[tls]
+enable = false
+
+[remote]
+port = 80
+
```

Comment:
Disable the control service:

    [control]
    enable = false

I tested these changes in
https://github.com/nirs/ovirt-imageio/commit/0a285564725f96f8f3baf56bb941cbaca6bd6fbe